From eba371974dc9343740ff3b3fd18caab8529291d1 Mon Sep 17 00:00:00 2001
From: openEuler Buildteam <buildteam@openeuler.org>
Date: Thu, 2 Jan 2020 15:34:13 +0800
Subject: [PATCH] device-mapper-multipath: fix find multipath failure

when DEFAULT_FIND_MULTIPATHS is FIND_MULTIPATHS_STRICT, it fail to
find multipath in find_multipaths function, use FIND_MULTIPATHS_OFF
instead of FIND_MULTIPATHS_STRICT.
---
 libmultipath/defaults.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/libmultipath/defaults.h b/libmultipath/defaults.h
index 7f3839f..c84d31d 100644
--- a/libmultipath/defaults.h
+++ b/libmultipath/defaults.h
@@ -17,7 +17,7 @@
 #define DEFAULT_NO_PATH_RETRY	NO_PATH_RETRY_UNDEF
 #define DEFAULT_VERBOSITY	2
 #define DEFAULT_REASSIGN_MAPS	0
-#define DEFAULT_FIND_MULTIPATHS	FIND_MULTIPATHS_STRICT
+#define DEFAULT_FIND_MULTIPATHS	FIND_MULTIPATHS_OFF
 #define DEFAULT_FAST_IO_FAIL	5
 #define DEFAULT_DEV_LOSS_TMO	600
 #define DEFAULT_RETAIN_HWHANDLER RETAIN_HWHANDLER_ON
-- 
1.8.3.1

